### PR TITLE
Passing MBus clock frequency to SimDRAM

### PR DESCRIPTION
**Pull request by jerryz123**
**Related issue**: 

**Type of change**: bug fix

### CODE REVIEW

File: generators/chipyard/src/main/scala/config/AbstractConfig.scala [hunk 1]
```diff
@@ -45,6 +45,8 @@ class AbstractConfig extends Config(
   new chipyard.config.WithNoSubsystemDrivenClocks ++             // drive the subsystem diplomatic clocks from ChipTop instead of using implicit clocks
   new chipyard.config.WithInheritBusFrequencyAssignments ++      // Unspecified clocks within a bus will receive the bus frequency if set
   new chipyard.config.WithPeripheryBusFrequencyAsDefault ++      // Unspecified frequencies with match the pbus frequency (which is always set)
+  new chipyard.config.WithMemoryBusFrequency(100.0) ++           // Default 100 MHz mbus
```
Comment (colinschmidt) [inline generators/chipyard/src/main/scala/config/AbstractConfig.scala:48]: Does this affect the default config's simulation speed?

Comment (jerryz123) [reply generators/chipyard/src/main/scala/config/AbstractConfig.scala:48]: No. The default was already 100 MHz here, this just makes it more explicit.

Comment (jerryz123) [reply generators/chipyard/src/main/scala/config/AbstractConfig.scala:48]: The explicitness is necessary for the WithSimDRAM HarnessBinder to pick it up

**Review by davidbiancolin — APPROVED** [review]
Ideally there'd be a clock sink in the port you could query, instead of grabbing from p(MemoryBusKey), but i know that's easier said than done. 

LGTM

**Pull request merged**